lib/gis_robot_suite/vector_normalizer.rb

```diff
@@ -29,18 +29,6 @@ def cleanup
       FileUtils.rm_rf tmpdir
     end
 
-    def run_shp2pgsql(projection, encoding, shpfn, schema, sqlfn, errfn)
-      # XXX: Perhaps put the .sql data into the content directory as .zip for derivative
-      # XXX: -G for the geography column causes some issues with GeoServer
-      cmd = "shp2pgsql -s #{projection} -d -D -I -W #{encoding} " \
-            "'#{shpfn}' #{schema}.#{bare_druid} " \
-            "> '#{sqlfn}' 2> '#{errfn}'"
-      logger.debug "Running: #{cmd}"
-      success = system(cmd, exception: true)
-      raise "normalize-vector: #{bare_druid} cannot convert Shapefile to PostGIS: #{File.open(errfn).readlines}" unless success
-      raise "normalize-vector: #{bare_druid} shp2pgsql generated no SQL?" unless File.size?(sqlfn)
-    end
```

Comment:
per the commit message, this only gets used in `LoadVector`, and all the info it needs is passed in via params (i.e. it doesn't use any instance methods from the surrounding class).  moving it back to its original home (it used to be in `LoadVector`) seemed like it'd be easier to maintain.